.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: ruby
+
+os:
+  - linux
+
+services:
+  - docker
+
+script:
+  - docker build -t lighthouse .
+  - ruby run.rb . example.com http://example.com
```

Comment:
Is Scriptster preinstalled?

Reply:
Nope 😞 

Just fixed in later commit https://github.com/iFixit/lighthouse-docker/pull/7/commits/e1ab89a44fb400e0ac0b4fea3a4502e3704f6b30